api-specificatie/Bevraging-Historie/openapi.yaml

```diff
@@ -857,70 +690,38 @@ components:
-          title: "Voornamen"
-          description: "<body>De verzameling namen die, gescheiden door spaties, aan\
-            \ de geslachtsnaam voorafgaat.</body><body><p>Element 02.10 De verzameling\
-            \ namen die, gescheiden door spaties, aan de geslachtsnaam voorafgaat.</p></body>"
-          maxLength: 200
-          example: "Pieter Jan"
-        voorvoegsel:
+          title: "adresRegel2"
+          description: "<body><p>Het tweede deel van het adres in het buitenland dat\
+            \ de ingeschreven persoon opgeeft bij vertrek naar het buitenland dan\
+            \ wel waar de ingeschreven persoon in het buitenland verblijft.</p></body>"
+          maxLength: 35
+        adresRegel3:
           type: "string"
-          title: "Voorvoegsel"
-          description: "<body>Dat deel van de geslachtsnaam dat voorkomt in de Voorvoegseltabel\
-            \ en, gescheiden door een spatie, vooraf gaat aan de rest van de geslachtsnaam.</body><body><p>Element\
-            \ 02.30</p></body>"
-          maxLength: 10
-          example: "de"
-        inOnderzoek:
-          $ref: "#/components/schemas/NaamInOnderzoek"
-    NaamInOnderzoek:
-      type: "object"
-      description: "<body><p>In onderzoekgroep om aan te geven of gegevens over de\
-        \ naam van de Ingeschreven persoon in onderzoek zijn.</p><p>Zie de <a>functionele\
-        \ specificaties</a> </p></body>"
-      properties:
-        geslachtsnaam:
+          title: "adresRegel3"
+          description: "<body><p>Het derde deel van het adres in het buitenland dat\
+            \ de ingeschreven persoon opgeeft bij vertrek naar het buitenland dan\
+            \ wel waar de ingeschreven persoon in het buitenland verblijft.</p></body>"
+          maxLength: 35
+        vertrokkenOnbekendWaarheen:
           type: "boolean"
-          title: "geslachtsnaam"
-          description: "<body><p>Indicator die aangeeft of het corresponderende gegeven\
-            \ voor deze persoon in onderzoek is.</p></body>"
-        voornamen:
-          type: "boolean"
-          title: "voornamen"
-          description: "<body><p>Indicator die aangeeft of het corresponderende gegeven\
-            \ voor deze persoon in onderzoek is.</p></body>"
-        voorvoegsel:
-          type: "boolean"
-          title: "voorvoegsel"
-          description: "<body><p>Indicator die aangeeft of het corresponderende gegeven\
-            \ voor deze persoon in onderzoek is.</p></body>"
-        datumEindeOnderzoek:
-          $ref: "#/components/schemas/Datum_onvolledig"
-        datumIngangOnderzoek:
-          $ref: "#/components/schemas/Datum_onvolledig"
+          title: "vertrokkenOnbekendWaarheen"
+          description: "<body><p>Indicatie dat de ingeschreven persoon is vertrokken\
+            \ naar het buitenland, maar dat niet bekend is waar naar toe.</p></body>"
+        land:
+          $ref: "#/components/schemas/Waardetabel"
     VerblijfplaatsInOnderzoek:
```

Comment:
Ik mis hier de datumeindeinonderzoek

Reply:
Er is een design decision (ik check of die gedocumenteerd is) dat we alleen actuele onderzoeken leveren. Er wordt dus geen historie op "inOnderzoek" getoond. Dan heb je dus nergens een einddatumInOnderzoek nodig.

Reply:
@JohanBoer  @fsamwel 
Betekent dit dan dat we nooit in de historie API InOnderzoek gegevens gaan tonen? We tonen namelijk niet actuele gegevens (hooguit met uitzondering van partner) maar gegevens uit de historie. 

Begrijp ik het zo goed? Zo ja, waarom hebben we dan nog InOnderzoek in de historie API?

Reply:
@KayodeBakker er is een verschil tussen een historisch gegeven dat in onderzoek is of een onderzoek dat zelf historisch is.

Wat wel mogelijk is, is dat onderzoek gedaan wordt naar een oud adres. Woonde de persoon vorig jaar wel op dat adres? het adres is historisch (de persoon woont inmiddels ergens anders), maar het onderzoek loopt nu nog wel. Dat tonen we dus ook, want de gebruiker moet weten dat er twijfels zijn over de betrouwbaarheid van dat vorige adres.

Wat we niet tonen is een onderzoek dat is afgerond, dus een einddatum heeft. Want de gebruiker hoeft niet te weten dat het adres ooit onderzocht is.

Reply:
> Wat wel mogelijk is, is dat onderzoek gedaan wordt naar een oud adres. Woonde de persoon vorig jaar wel op dat adres? het adres is historisch (de persoon woont inmiddels ergens anders), maar het onderzoek loopt nu nog wel. Dat tonen we dus ook, want de gebruiker moet weten dat er twijfels zijn over de betrouwbaarheid van dat vorige adres.

Wat we niet tonen is een onderzoek dat is afgerond, dus een einddatum heeft. Want de gebruiker hoeft niet te weten dat het adres ooit onderzocht is.

Wellicht dat ik je verkeerd begrijp, maar dat klonk tegenstrijdig. We tonen de gebruiker dat een historisch adres ooit in onderzoek heeft gestaan, maar we tonen niet de einddatum want de gebruiker hoeft niet te weten dat een historisch adres ooit in onderzoek heeft gestaan?

Maar los van de functionele verwarring die ik er nu over heb, vraag ik me af hoe we dit qua implementatie moeten zien. Betekent dit dat we voor partners, verblijfplaatsen en verblijfstitels  alles van InOnderzoek mappen op de eind datum na? Ook als deze er wel staat?

Reply:
> Betekent dit dan dat we nooit in de historie API InOnderzoek gegevens gaan tonen? We tonen namelijk niet actuele gegevens (hooguit met uitzondering van partner) maar gegevens uit de historie.

Dit is voor mij een nieuw inzicht. Is het zo dat de huidige verblijfplaats van een persoon dan niet opgenomen wordt in de verblijfplaatshistorie ? Dat lijkt mij niet de bedoeling. En ik kan mij bij bewoning ook heel goed voorstellen dat je alle huidige bewoners ook wilt zien, bij partners dat je alle huidige partners ook wilt zien.

Waar ik gisteren wonde is toch ook historie, maar die zou niet worden getoond omdat dat gelijk is aan de actuele situatie ? 

Ik begrijp nu wel jouw verwarring over inOnderzoek.

Reply:
>Dit is voor mij een nieuw inzicht. Is het zo dat de huidige verblijfplaats van een persoon dan niet opgenomen wordt in de verblijfplaatshistorie ? Dat lijkt mij niet de bedoeling. En ik kan mij bij bewoning ook heel goed voorstellen dat je alle huidige bewoners ook wilt zien, bij partners dat je alle huidige partners ook wilt zien.

Ik zou verwachten dat voor bewoners wellicht het nog wel interessant is, maar in een historie api lijkt me niet logisch om ook de actuele verblijfplaats en verblijfstitel weer te geven. Dat we dit bij Partner doen is omdat een ontbonden partner als historie wordt gezien, maar deze wordt door de GBA-V opgenomen als actueel. De huidig niet ontbonden partner tonen we bijv. dan weer niet. De api was naar mijn weten ook puur alleen voor historie en niet voor actuele gegevens dat niet als historisch ervaren kan worden (zoals dus ontbonden partners).

Reply:
> Wellicht dat ik je verkeerd begrijp, maar dat klonk tegenstrijdig. We tonen de gebruiker dat een historisch adres ooit in onderzoek heeft gestaan, maar we tonen niet de einddatum want de gebruiker hoeft niet te weten dat een historisch adres ooit in onderzoek heeft gestaan?
> 
@KayodeBakker we tonen dat het historische adres NU in onderzoek is. We tonen niet dat het adres ooit in onderzoek is geweest (onderzoek is inmiddels afgerond). Een onderzoek met een einddatum tonen we dus nooit. 
Oftewel: we tonen geen historische onderzoeken, maar wel historische adressen die nu in onderzoek zijn. Bijvoorbeeld om te achterhalen of iemand destijds rechtmatig een uitkering heeft ontvangen.

Reply:
@CathyDingemanse 
> we tonen dat het historische adres NU in onderzoek is. ...

Dus als een historisch bijv. adres inonderzoek is, maar dat inonderzoek heeft een einddatum dan nemen we wel dat historisch adres op, maar is dan heel inonderzoek van dat historisch adres niet getoond OF alleen de einddatum?
Welke van de 2 het ook wordt, wat is de reden dat dit niet wordt getoond? Als het relevant is om te zien dat iets inonderzoek staat dan is het toch ook relevant om te zien dat deze inonderzoek heeft gestaan? Vanuitgaand dat we inonderzoek wel tonen, maar niet de einddatum, dan wekt dat toch ook de indruk op dat deze nog steeds inonderzoek is?

Reply:
> Dus als een historisch bijv. adres inonderzoek is, maar dat inonderzoek heeft een einddatum dan nemen we wel dat historisch adres op, maar is dan heel inonderzoek van dat historisch adres niet getoond OF alleen de einddatum?

als een inOnderzoek een einddatum heeft, wordt dat onderzoek NIET opgenomen. We nemen alleen inOnderzoek op wanneer dat op dit moment nog loopt. Ook wanneer dat onderzoek gaat over een adres dat historisch is.

> Welke van de 2 het ook wordt, wat is de reden dat dit niet wordt getoond? Als het relevant is om te zien dat iets inonderzoek staat dan is het toch ook relevant om te zien dat deze inonderzoek heeft gestaan? Vanuitgaand dat we inonderzoek wel tonen, maar niet de einddatum, dan wekt dat toch ook de indruk op dat deze nog steeds inonderzoek is?

Wanneer een onderzoek is afgerond kunnen er grofweg twee dingen gebeuren:
1. uit het onderzoek is gebleken dat de registratie correct is.
2. uit het onderzoek is gebleken dat de registratie onjuist is.

Aangezien we nooit historische voorkomens opnemen die onjuist zijn (GBA-V geeft die ook niet terug), weten we zeker dat elk afgerond onderzoek als conclusie had dat er niks aan de hand was.
We nemen in onderzoek alleen op om de gebruiker erop te wijzen dat er twijfels zijn over de juistheid van de getoonde gegevens. Wanneer jij een afgerond onderzoek ziet, zijn er geen twijfels meer over de juistheid en nemen we dat dus niet op.